src/Uno.UI/Mock/ArbitraryShapeBase.reference.cs

```diff
@@ -9,8 +9,5 @@ namespace Windows.UI.Xaml.Shapes
 {
 	public abstract partial class ArbitraryShapeBase : Shape
```

Comment:
I think that WASM is the only one which is still using that class, you can you just remove it (it's not part of the WinUI contract)

Also you should be able to remove all the `#if LEGACY_SHAPE_MEASURE` code.

Reply:
Yup this PR was a small first cleanup step. Will try cleanup everything is upcoming PR(s).